ultravox/data/datasets.py

```diff
@@ -438,6 +440,7 @@ def _get_sample(self, idx: int, row: transformers.BatchFeature) -> VoiceSample:
         return VoiceSample(
             self._get_transcribe_messages(idx, audio_transcript),
             self._get_audio(row),
+            audio_transcript=audio_transcript,
```

Comment:
Simple bugfix: `boolq_in` didn't have `audio_transcript`.

Reply:
This should probably use `_get_transcribe_messages` which sidesteps this issue